src/plugins/vis_type_timeseries/public/application/components/aggs/histogram_support.test.js

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
nit: we already started migration` js -> ts`. If possible please refactor tests in `ts`

Reply:
@alexwizp are all of the imported files already migrated to TS? Maybe we can hold off here for now if not?